Testscripts/Linux/utils.sh

```diff
@@ -1963,23 +1963,52 @@ function get_host_version() {
 
 # Validate the exit status of previous execution
 function check_exit_status() {
+	# The failed/aborted options are used when Linux script is the testscript,
+	# to check the checkpoint and set test result.
 	exit_status=$?
 	message=$1
+	test_state=$2
+	failed_message=$3
```

Comment:
Is this additional failure message required ? I could see it is not used in most of the cases.

Reply:
Hi @johnsongeorge-w ,
Thank you for your feedback! I just want to log some error output of the command somewhere if it fails. I think it can be logged in the <script_name>_summary.log but this file is not collected in the test result now. Any suggestion about that? Thanks!

Reply:
This will be supported soon. <script_name> _summary.log file will be collected in test results.

Reply:
Thank you @johnsongeorge-w ! Removed the failed_message.

Reply:
Thanks @johnsongeorge-w ! Removed the failed_message from the function.